cmd/worker/main.go

```diff
@@ -176,7 +176,7 @@ func handleMessage(ctx context.Context, msg *pubsub.Message, packagesBucket *blo
 		return err
 	}
 
-	var staticResults result.StaticAnalysisResults
+	var staticResults analysisrun.StaticAnalysisResults
```

Comment:
Yeah let's try and shorten this type name in another PR.

Reply:
SGTM.